app.js

```diff
@@ -0,0 +1,74 @@
+var arr = 
+[
+  {
+    name: "Lion King",
+    license: [2011, 2013]
+  },
+
+  {
+    name: "Iron Man",
+    license: [2005,2010]
+  },
+  {
+    name: "Life of Pi",
+    license: [2008,2014]
+  },
+  {
+    name: "Another Movie",
+    license: [2011,2013]
+  }
+];
+
+
+var frequency = {};
+
+function main(arr)
+{
+	countYearFrequency(arr)
```

Comment:
Avoid mutating the given array

If someone else on the team, or some other code is also using the array to compute some data, they have lost the original array. Try not to modify inputs, until absolutely necessary

Reply:
I don't think I am mutating the array. Am I missing something ?